client/platform/web-girder/views/ViewerAlert.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { defineComponent, ref, toRef } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'ViewerAlert',
+  setup(_, { root }) {
+    const dialog = ref(false);
+
+
+    // this will change once REST updates are merged
+    const brandData = toRef(root.$store.state.Brand, 'brandData');
+
+    return {
+      dialog,
+      brandData,
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- This component is for alert messages on the Viewer Page only.
+    See NavigationBar for sitewide alert banner -->
+  <v-dialog
+    v-model="dialog"
+    max-width="600px"
```

Comment:
In most of the other dialogs, we've added `overlay-opacity="0.90"` to make the background fade more while the dialog is open.  Could you add that here as well?